src/kernel/include/shared/cos_types.h

```diff
@@ -235,16 +236,17 @@ static inline unsigned long captbl_idsize(cap_t c)
  * 2GB-> = system physical memory
  */
 enum {
-	BOOT_CAPTBL_SRET       = 0,
-	BOOT_CAPTBL_SELF_CT    = 4,
-	BOOT_CAPTBL_SELF_PT    = 6,
-	BOOT_CAPTBL_SELF_COMP  = 8,
-	BOOT_CAPTBL_BOOTVM_PTE = 12,
-	BOOT_CAPTBL_PHYSM_PTE  = 14,
-	BOOT_CAPTBL_KM_PTE     = 16,
-
-	BOOT_CAPTBL_COMP0_CT           = 18,
-	BOOT_CAPTBL_COMP0_PT           = 20,
+	BOOT_CAPTBL_SRET          = 0,
+	BOOT_CAPTBL_SELF_CT       = 4,
+	BOOT_CAPTBL_SELF_PT       = 6,
+	BOOT_CAPTBL_SELF_COMP     = 8,
+	BOOT_CAPTBL_BOOTVM_PTE    = 12,
+	BOOT_CAPTBL_SELF_LOCAL_PT = 14,
```

Comment:
I'm missing the distinction of SELF_PT and UNTYPED_PT here, especially in case of Booter component. Isn't `pgtbl` of a component supposed to have all the pages that it has access to? How is separating it between two PT caps achieve this? Also, every time booter component or any component that has access to UNTYPED_PT, would then alloc, which will have to cpy from its UNTYPED_PT to SELF_PT to mean that it has access to its resources. 

One thing this simplifies I assume is, when we have a separate memmgr, we can just copy UNTYPED_PT cap to memmgr as opposed to copying as many pgtbl caps that it can manage resources for components, but that should not be much of a overhead if that happens only at boot-time. What am I missing here?